### PR TITLE
Fix code after review

### PR DESCRIPTION
**Pull request by legafise**

### CODE REVIEW

File: src/com/lashkevich/flowers/entity/SpikedStoreFlowers.java [hunk 1]
```diff
@@ -4,28 +4,28 @@
...
-    public SpikedStoreFlowers(long id, String name, double height, List<Color> color, double price, double freshness, boolean spikes, List<Accessory> accessories) {
+    public SpikedStoreFlowers(long id, String name, double height, List<Color> color, double price, double freshness, boolean hasSpikes, List<Accessory> accessories) {
         super(id, name, height, color, price, freshness, accessories);
-        this.spikes = spikes;
+        this.hasSpikes = hasSpikes;
     }
 
-    public boolean isSpikes() {
-        return spikes;
+    public boolean isHasSpikes() {
```
Comment (IvanYarochkin) [inline src/com/lashkevich/flowers/entity/SpikedStoreFlowers.java]: Name of this method should be just `hasSpikes()`

File: src/com/lashkevich/flowers/util/checker/FlowersUnderSnowInBouquetChecker.java [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+
+public class FlowersUnderSnowInBouquetChecker {
+    public static Flower checkFlower(Flower flower) {
+        if (flower instanceof SpringFlower) {
+            SpringFlower springFlower = (SpringFlower) flower;
+            if (springFlower.hasSeem()) {
+                return springFlower;
+            } else {
+                return GeneralStoreFlowerRandomizer.generateRandomStoreFlower();
+            }
```
Comment (IvanYarochkin) [inline src/com/lashkevich/flowers/util/checker/FlowersUnderSnowInBouquetChecker.java]: Move this code to another private method in the same class.

**Review by IvanYarochkin — CHANGES_REQUESTED** [review]

File: src/com/lashkevich/flowers/entity/SpikedStoreFlowers.java (hunk 1 above)
Comment (legafise) [reply src/com/lashkevich/flowers/entity/SpikedStoreFlowers.java]: Done

File: src/com/lashkevich/flowers/util/checker/FlowersUnderSnowInBouquetChecker.java (hunk 2 above)
Comment (legafise) [reply src/com/lashkevich/flowers/util/checker/FlowersUnderSnowInBouquetChecker.java]: Done

File: src/com/lashkevich/flowers/util/checker/FlowersUnderSnowInBouquetChecker.java [hunk 3]
```diff
@@ -7,14 +7,18 @@
...
-            } else {
-                return GeneralStoreFlowerRandomizer.generateRandomStoreFlower();
-            }
+            return flowersSubstitute(flower);
         } else {
             return flower;
         }
     }
+
+    private static Flower flowersSubstitute(Flower flower) {
```
Comment (IvanYarochkin) [inline src/com/lashkevich/flowers/util/checker/FlowersUnderSnowInBouquetChecker.java]: A name of a method should be started from the verb.

**Review by IvanYarochkin — CHANGES_REQUESTED** [review]

File: src/com/lashkevich/flowers/util/checker/FlowersUnderSnowInBouquetChecker.java (hunk 3 above)
Comment (legafise) [reply src/com/lashkevich/flowers/util/checker/FlowersUnderSnowInBouquetChecker.java]: Done

**Pull request merged**